src/lib.rs

```diff
@@ -133,26 +138,71 @@ impl From<io::Error> for ChunkSaveError {
     }
 }
 
-pub struct AnvilChunkProvider<'a> {
+#[derive(Copy, Clone, Debug, Eq, PartialEq)]
+pub struct RegionAndOffset {
```

Comment:
1. I think this structure is not neccesery. 
2. I think it should have diffrent name. It's about coordinates. Something like `RegionCoord` ?
3. Should be private?